thesis/testing.tex

```diff
@@ -67,3 +71,16 @@ \section{Continuous Integration}
 When a pull request is reviewed, it contains information about the test results and what the code coverage is.
 If code coverage decreases in the pull request, compared to the master branch, then the developer probably did not test the code well enough. 
 The lint check helps the reviewer to focus on the functional parts instead of syntactic and code style things.
+
+\section{Conclusion}
+On the back-end the test setup was easy to make working.
+The \glsi{node} community have lots of libraries and tools to do this and there are many resources on how to set it up.
+
+On the front-end, the test setup was, at first glance, easy to set up, but in the process we found out that this was actually not the case.
+This lead us to decide if we wanted no tests at all or some representative tests to test core logic.
```

Comment:
"this lead us to decide" .. i dont think that is correct.   maybe something like "duo to this we had to make a choice..."